app/components/loadMoreNote.tsx

```diff
@@ -0,0 +1,31 @@
+import { TimelineButton } from "~/components/timelineButton";
+import style from "./loadMoreNote.module.css";
+
+export interface LoadMoreNoteButtonProps {
+  type: "newer" | "older";
+  noteID: string;
+}
+
+export const LoadMoreNoteButton = (props: LoadMoreNoteButtonProps) => {
+  if (props.type === "newer") {
+    return (
+      <div className={style.loadMoreNote}>
+        <TimelineButton
+          link={`?after_id=${props.noteID}#`}
+          linkText="Load newer notes"
+        />
+      </div>
+    );
+  } else if (props.type === "older") {
+    return (
+      <div className={style.loadMoreNote}>
+        <TimelineButton
+          link={`?before_id=${props.noteID}#`}
+          linkText="Load older notes"
+        />
+      </div>
+    );
+  } else {
+    <></>;
```

Comment:
```suggestion
    throw new Error("unreachable");
```
空行にするのがコードスタイル的に問題なのであれば, ありえないコードパスなので例外を投げてみてはどうでしょう.